src/components/graph/graph.renderer.jsx

```diff
@@ -105,9 +105,17 @@ function _renderDefs() {
             return cachedDefs;
         }
 
-        const small = MARKER_SMALL_SIZE;
-        const medium = small + (MARKER_MEDIUM_OFFSET * config.maxZoom) / 3;
-        const large = small + (MARKER_LARGE_OFFSET * config.maxZoom) / 3;
+        var small = MARKER_SMALL_SIZE;
+        var medium = small + (MARKER_MEDIUM_OFFSET * config.maxZoom) / 3;
+        var large = small + (MARKER_LARGE_OFFSET * config.maxZoom) / 3;
+
+        switch (config.node.symbolType) {
```

Comment:
Please move this onto a function. It can return an object with the values for `small`, `medium` and `large` markers sizes.